.github/actions/process_recipe/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Process Recipe'
+description: '''
+For a given recipe.py and its meta.yaml file: convert it to a Prefect Flow,
+register the Flow with the specified Bakery in the meta.yml, and test a pruned run
+of the Flow on the Bakery.
+'''
+inputs:
+  path_to_bakeries_yaml:
+    description: 'Path to the bakeries.yaml'
+    required: true
```

Comment:
Do we want to have a default which points to the latest bakery database?

Is a url acceptable here?

Reply:
This input references the latest bakery database which is downloaded in a previous step in the workflow https://github.com/pangeo-forge/staged-recipes/pull/28/files#diff-7e3164688a46495c98f6e332d8be1f3f0b64a4b2c8389dceebf55e4532185aefR17